src/docs/product/security/ip-ranges.mdx

```diff
@@ -14,12 +14,32 @@ apply to Self-hosted or Single Tenant.
 </markdown>
 </Alert>
 
-Sentry is served from a single IP address for all web traffic and all event accepting. The IP address is:
+## Dashboard and API
+
+Sentry's dashboard and API are served from a single IP address for all web traffic. The IP address is:
 
 ```python
 35.188.42.15/32
 ```
 
+## Event Ingestion
+
+Sentry's Event Ingestion respects two domains within a Data Source Name (DSN): [Organization Subdomains](https://forum.sentry.io/t/organization-subdomains-in-dsns/9360) and Sentry's apex domain.
+
+Sentry's primary domain accepts events from the same IP address as the Dashboard and API:
+
+```python
+35.188.42.15/32
+```
+
+Starting $DATE, Sentry's Organization Subdomains will migrate from the prior IP address to the following single address:
```

Comment:
```suggestion
Starting $DATE, Sentry's organization subdomains will migrate from the prior IP address to the following single address:
```

Reply:
Do we want to explain that there's no impact to existing customers? "This will not require any action for existing customers."

Reply:
The action required will be announced via blogpost/broadcast like [this one](https://blog.sentry.io/2017/06/15/notice-of-address-change).

Reply:
👍